elyra/pipeline/processor_local.py

```diff
@@ -23,6 +23,7 @@
 import time
 from typing import Dict
 from typing import List
+from typing import Optional
```

Comment:
`sys` is needed for the `sys.executable` reference below.
```suggestion
from typing import Optional
import sys
```